src/components/Graph/Graph.tsx

```diff
@@ -215,6 +222,17 @@ const GraphMemo: FC<GraphMemoProps> = memo(
       >
         {nodeCoordinates !== undefined
           ? Object.keys(nodeCoordinates).map(nodeName => {
+              let childOfCurrent = false;
```

Comment:
Another naming comment - to stay consistent with current naming conventions this would be if the node is a transition of the current node so I would rename to `transitionOfCurrent`